maro/cli/grass/executors/grass_executor.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, cluster_details: dict):
 
     def remote_build_image(self, remote_context_path: str, remote_image_name: str):
         command = (
-            f"ssh -o StrictHostKeyChecking=no "
+            "ssh -o StrictHostKeyChecking=no "
```

Comment:
Since "ssh -o StrictHostKeyChecking=no " has been reused for too many times, it supposed could be declared as a  constant string.

Reply:
Prefer not to do it, which hurts the readability of the code